### PR TITLE
Feature: Import/Export Wallet

### PR DESCRIPTION
**Pull request by TimovanDijk**
Added import export wallet support. Added Android support and using existing IOS support

### CODE REVIEW

File: src/index.js
Comment (jakubkoci) [inline src/index.js:295]: Could you please throw an Error for the iOS platform? The same for the second method.

```suggestion
    if (Platform.OS === 'ios') {
      throw new Error(`Unsupported operation! Platform: ${Platform.OS}`)
    }

    return IndySdk.exportWallet(wh, JSON.stringify(exportConfig))
```

**Review by jakubkoci — CHANGES_REQUESTED** [review]
Hi, @TimovanDijk. Thanks for your contribution. Could you just add the condition as I described. It will be easier to recognize what is not implemented on the iOS platform. Thanks.

File: src/index.js
Comment (TimoGlastra) [reply src/index.js:295]: @jakubkoci It is already implemented for iOS.

@TimovanDijk can't test on iOS, so I'll test it on iOS and report back

**Comment by TimoGlastra** [comment]
I still need to look at it! Can you reopen?

**Comment by jakubkoci** [comment]
Oh no. I'm sorry. I did a force push with sign-off for each of my commits to merge it to the new Hyperledger repo 🤦‍♂️  I cannot reopen the PR.

We should perhaps do a new PR there when I commit the changes from this repo to the Hyperledger one. I have everything prepared, waiting for an official approval now.

Hopefully, the changes should be available here https://github.com/AbsaOSS/rn-indy-sdk/commit/bb75a2bcd4342d044058fa96ba5ddd4abe52a129

**Comment by TimoGlastra** [comment]
I'll reopen when the transfer is complete

**Pull request closed without merging**
